### ISSUE TITLE
trello

### ISSUE THREAD

**Issue by progrium** [tractordev/integra#14]
https://github.com/APIs-guru/openapi-directory/blob/main/APIs/trello.com/1.0/openapi.yaml

**Comment by lritter79** [comment]
This is trello's offical API, but it's in JSON:https://dac-static.atlassian.com/cloud/trello/swagger.v3.json?_v=1.576.0

Now, what i can do is convert it from JSON to YAML, and run that instead of the API Guru's version. Thoughts?

**Comment by progrium** [comment]
Yeah, it's hard to tell but the official one looks newer. I think we support json files now but I'd rather have formatted YAML in the repo when possible, so yeah try converting it.
